packages/unit-testing/stubs/workbook/DataDimensionItemValue.js

```diff
@@ -0,0 +1,48 @@
+define(['./DataDimension', './Record', './Currency', './Range'], function (DataDimension, Record, Currency, Range) {
+	/**
+	 * @class DataDimensionItemValue
+	 * @classDescription Class holding reference to the dimension item and its value
+	 * @constructor
+	 * @protected
+	 *
+	 * @since 2021.2
+	 */
+	function DataDimensionItemValue() {
+		/**
+		 * Data dimension item
+		 * @name DataDimensionItemValue#dataDimensionItem
+		 * @type {DataDimension}
+		 * @throws {SuiteScriptError} READ_ONLY_PROPERTY when setting the property is attempted
+		 *
+		 * @since 2021.2
+		 */
+		this.item = undefined;
+		/**
+		 * Value of the data dimension item
+		 * @name DataDimensionItemValue#value
+		 * @type {string|number|boolean|Record|Currency|Range}
```

Comment:
In helps also appears workbook.Durantion a possible return type. Let's leave it as it is. No action needed.